src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -145,6 +144,7 @@ export function ConfirmationModal ({ transaction, handleClose, declineTransactio
   const { id, counterparty, amount, type } = transaction
 
   const action = type === TYPE.request ? 'pay' : 'decline'
+  const contentLabel = `${_.capitalize(action)} ${type}?`
```

Comment:
I meant that we should set this in the switch statement below, and not capitalize the action. So there would be two cases
```
contentLabel = 'Pay request'
```
and
```
contentLabel = `Reject ${type}?`
```

Reply:
No problem; this is an easy update.  Why recommend this approach?

Reply:
``${_.capitalize(action)} ${type}?`` this requires a bunch of thinking to read, and doesn't really buy us anything.

Reply:
@JettTech but having made the suggestion, I'm happy to defer to you. If you want to merge as is, go ahead.